
2022-10-25 00:28:53.164 - [Debug] [Connection] Attacker connected: 222.134.240.91 | Client Identification: SSH-2.0-Go
2022-10-25 00:28:53.172 - [Debug] [Auth] Attacker 147.182.233.56 trying to authenticate with 'password'
2022-10-25 00:28:53.172 - [Debug] [Auto Access] Attacker: 147.182.233.56, Threshold: 1, Attempts: 1
2022-10-25 00:28:53.173 - [Debug] [Auto Access] Compromising the honeypot
2022-10-25 00:28:53.173 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-25 00:28:54.732 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-25 00:28:54.733 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 00:28:54.733 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.117
2022-10-25 00:28:54.751 - [Debug] [Auth] Attacker 139.59.135.142 trying to authenticate with 'publickey'
2022-10-25 00:28:54.759 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 00:28:54.760 - [Debug] [Auth] Attacker: 139.59.135.142 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 00:28:54.763 - [Debug] [Auth] Attacker 223.171.91.120 trying to authenticate with 'none'
2022-10-25 00:28:54.769 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 00:28:54.780 - [Debug] [Connection] Attacker connected: 69.236.215.6 | Client Identification: SSH-2.0-Go
2022-10-25 00:28:54.810 - [Debug] [Connection] Attacker connected: 102.141.225.238 | Client Identification: SSH-2.0-Go
2022-10-25 00:28:54.818 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:1850226) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.45',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-25 00:30:10.721 - [Info] MITM Version: 2
2022-10-25 00:30:10.722 - [Info] Auto Access Enabled: true
2022-10-25 00:30:10.723 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 00:30:10.739 - [Info] SSH man-in-the-middle server for 10.0.3.45 listening on 10.0.3.1:1003
2022-10-25 00:30:10.956 - [Debug] [Connection] Attacker connected: 78.189.25.224 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:10.980 - [Debug] [Connection] Attacker connected: 43.139.28.174 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:11.207 - [Debug] [Connection] Attacker connected: 223.171.91.158 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:11.958 - [Debug] [Connection] Attacker connected: 85.105.82.39 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:12.265 - [Debug] [Connection] Attacker connected: 85.53.55.133 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:12.344 - [Debug] [Auth] Attacker 43.139.28.174 trying to authenticate with 'none'
2022-10-25 00:30:12.345 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 00:30:12.531 - [Debug] [Auth] Attacker 78.189.25.224 trying to authenticate with 'none'
2022-10-25 00:30:12.532 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 00:30:12.737 - [Debug] [Auth] Attacker 223.171.91.158 trying to authenticate with 'none'
2022-10-25 00:30:12.737 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 00:30:12.827 - [Debug] [Auth] Attacker 43.139.28.174 trying to authenticate with 'publickey'
2022-10-25 00:30:12.829 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 00:30:12.829 - [Debug] [Auth] Attacker: 43.139.28.174 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 00:30:12.834 - [Debug] [Auth] Attacker 85.105.82.39 trying to authenticate with 'none'
2022-10-25 00:30:12.834 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 00:30:12.922 - [Debug] [Auth] Attacker 85.53.55.133 trying to authenticate with 'none'
2022-10-25 00:30:12.922 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 00:30:12.974 - [Debug] [Auth] Attacker 78.189.25.224 trying to authenticate with 'password'
2022-10-25 00:30:12.974 - [Debug] [Auto Access] Attacker: 78.189.25.224, Threshold: 1, Attempts: 1
2022-10-25 00:30:12.974 - [Debug] [Auto Access] Compromising the honeypot
2022-10-25 00:30:12.975 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-25 00:30:13.050 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-25 00:30:13.052 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 00:30:13.052 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.45
2022-10-25 00:30:13.103 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-25 00:30:13.113 - [Debug] [Auth] Attacker 85.105.82.39 trying to authenticate with 'publickey'
2022-10-25 00:30:13.114 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 00:30:13.114 - [Debug] [Auth] Attacker: 85.105.82.39 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 00:30:13.122 - [Debug] [Auth] Attacker 85.53.55.133 trying to authenticate with 'publickey'
2022-10-25 00:30:13.123 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 00:30:13.123 - [Debug] [Auth] Attacker: 85.53.55.133 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 00:30:13.305 - [Debug] [Auth] Attacker 223.171.91.158 trying to authenticate with 'password'
2022-10-25 00:30:13.306 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 00:30:13.306 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.45
2022-10-25 00:30:13.313 - [Debug] [Connection] Attacker closed the connection
2022-10-25 00:30:13.329 - [Debug] [Connection] Attacker closed the connection
2022-10-25 00:30:13.347 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-25 00:30:13.390 - [Debug] [Connection] Attacker closed the connection
2022-10-25 00:30:13.441 - [Debug] [Connection] Attacker connected: 85.53.55.133 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:13.519 - [Debug] [Connection] Attacker connected: 85.105.82.39 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:13.540 - [Debug] [Connection] Attacker connected: 43.139.28.174 | Client Identification: SSH-2.0-Go
2022-10-25 00:30:13.876 - [Debug] [EXEC] Noninteractive mode attacker command: PATH=/dev/shm/:/tmp/:./:/var/tmp/:/root/:/etc/:$PATH nc localhost 1234
